.github/workflows/auto-close-pull-request.yml

```
@@ -11,7 +11,13 @@ permissions: {}
 jobs:
   auto-close-dependency-pull-request:
     name: Auto close non-maintainer dependency pull request
-    if: ${{ !contains(fromJSON('["MEMBER", "OWNER"]'), github.event.pull_request.author_association) && github.event.pull_request.user.type != 'Bot' && !github.event.pull_request.draft }}
+    if: >
+      ${{ 
+        !contains(fromJSON('["MEMBER", "OWNER"]'), github.event.pull_request.author_association) && 
+        github.event.pull_request.user.type != 'Bot' && 
+        !github.event.pull_request.draft &&
+        github.event.pull_request.labels.*.name != 'allow-dependency-change'
```

```suggestion
        !contains(github.event.pull_request.labels.*.name, 'allow-dependency-change')
```

https://docs.github.com/en/actions/learn-github-actions/expressions#example-using-an-object-filter

---

Thank you, I was unsure about how to test this 🙇